assets/css/style.css

```diff
@@ -0,0 +1,5 @@
+h1 {
+  border: 1px solid blue;
+  box-shadow: 0 0 5px red;
+  background-color: silver;
+}
```

Comment:
Marketing wants this to be center aligned and also 50px font.

Reply:
Good catch.